src/client/components/ModalViewComponent/membersData.json

```diff
@@ -2,25 +2,31 @@
   {
     "userId": 1,
     "role": "basic",
```

Comment:
The data should come from the database instead

Reply:
But there won't be any picture.

Reply:
The data should come from the database. Then you can edit your database if you want to send back images

Reply:
Its in the components folder. My fault.